lib/src/npm.dart

```diff
@@ -296,6 +297,24 @@ String get _wrapperLibrary {
     wrapper.writeln("import $target as module_main;");
   }
 
+  // Define a JS-interop Future to Promise translator so that we can export
+  // a Promise-based API
+  wrapper.writeln("""
+Promise _futureToPromise(Future future) {
+  return new Promise(allowInterop((resolve, reject) {
+    future.then(resolve, onError: reject);
+  }));
+}
```

Comment:
Why not use [`futureToPromise()`](https://pub.dev/documentation/node_interop/latest/node_interop.util/futureToPromise.html) from `node_interop`?

Reply:
Somehow, I missed finding that. (done)